tools/lsp/completion.rs

```diff
@@ -301,14 +300,17 @@ pub(crate) fn completion_at(
     } else if node.kind() == SyntaxKind::Document {
         let r: Vec<_> = [
             // the $1 is first in the quote so the filename can be completed before the import names
-            ("import", "import { $2 } from \"$1\";"),
-            ("component", "component $1 {}"),
-            ("struct", "struct $1 {}"),
-            ("global", "global $1 {}"),
-            ("export", "export { $1 }"),
-            ("export component", "export component $1 { }"),
-            ("export struct", "export struct $1 {}"),
-            ("export global", "export global $1 {}"),
+            ("import", "import { $1 } from \"${2:std-widgets.slint}\";"),
+            ("component", "component ${1:Component} ${2:inherits ${3:Rectangle} }{\n    $0\n}"),
+            ("struct", "struct ${1:Name} {\n    $0\n}"),
+            ("global", "global ${1:Name} {\n    $0\n}"),
+            ("export", "export { $0 }"),
+            (
+                "export component",
+                "export component ${1:AppWindow} ${2:inherits ${3:Window} }{\n    $0\n}",
```

Comment:
Not sure we want to use AppWindow here. This is only hapening once in a project.  (also no inherits)

Reply:
> the name is not that bad

Yes, I think it's that bad.
It suggests that this is only for the main component and not for reusable ones.